app.js

```diff
@@ -79,6 +60,19 @@ var el = document.getElementById('imageContainer');
 
 var previousImages = [];
 
+function loadProductsFromLocalStorage() {
+  if (localStorage.allProducts) {
+    productImages = JSON.parse(localStorage.allProducts);
+  }
+}
+
+loadProductsFromLocalStorage();
+
+function saveProductsToLocalStorage(products){  // use productImages as that is my array that stores the images.
```

Comment:
I like this comment as a way to keep track of what will be passed through into the parameter of this function